tests/harness/cfg/tests.yaml

```diff
@@ -0,0 +1,164 @@
+# Test configurations specify which requirements are needed for the test to run.
+# These requirements refer to both software packages and ZenML stack components.
+#
+# Each entry must point to the pytest module or package that contains the test
+# code and specify a list of requirements that are needed for the test to run.
+# The requirements can be specified either inline using the same format as the
+# global requirements file or by referencing a global requirements entry by
+# name.
+#
+# Note that the integrations, packages and system tools requirements are not
+# installed automatically. The user is expected to install them in the host OS
+# and/or current virtual environment. The test framework only checks for their
+# presence and skips the test if they are not installed.
+#
+# Example:
+#
+#  ```
+#  tests:
+#    - module: tests.integration.example.module
+#      requirements:
+#        - global_requirement_name
+#        - integrations:
+#            - pytorch
+#            - xgboost
+#          packages:
+#            - numpy==1.19.5
+#            - pandas
+#          system_tools:
+#            - docker
+#            - kubectl
+#          system_os:
+#            - linux
+#            - macos
+#          stacks:
+#            - type: experiment_tracker
+#              flavor: mlflow
+#              configuration:
+#                nested: true
+#            - type: model_deployer
+#              flavor: seldon
+#  ```
+tests:
+
+  - module: tests.integration.functional.test_zen_server_api
+    requirements:
+      - capabilities:
+          server: true
+
+  - module: tests.integration.functional.secrets_manager
+    requirements:
+      - stacks:
+          - type: secrets_manager
+      - capabilities:
+          secrets-scoping: true
+
+  - module: tests.integration.functional.cli
```

Comment:
Ah, no, not really. I listed them here because I had them temporarily disabled while I was converting them into integration tests. But this is not needed anymore, just a left-over on my part. I'll remove it.